src/Data/Product/Relation/Binary/Lex/Strict.agda

```diff
@@ -134,24 +134,26 @@ module _ {_≈₁_ : Rel A ℓ₁} {_<₁_ : Rel A ℓ₂}
     antisym (inj₂ x≈≤y)  (inj₂ y≈≤x)  =
       proj₁ x≈≤y , antisym₂ (proj₂ x≈≤y) (proj₂ y≈≤x)
 
+  ×-respectsʳ : Transitive _≈₁_ →
+                _<₁_ Respectsʳ _≈₁_ → _<₂_ Respectsʳ _≈₂_ →
+                _<ₗₑₓ_ Respectsʳ _≋_
+  ×-respectsʳ trans resp₁ resp₂ y≈y' (inj₁ x₁<y₁) = inj₁ (resp₁ (proj₁ y≈y') x₁<y₁)
+  ×-respectsʳ trans resp₁ resp₂ y≈y' (inj₂ x≈<y)  = inj₂ (trans (proj₁ x≈<y) (proj₁ y≈y')
+                                                       , (resp₂ (proj₂ y≈y') (proj₂ x≈<y)))
+  ×-respectsˡ : IsPartialEquivalence _≈₁_ →
+                _<₁_ Respectsˡ _≈₁_ → _<₂_ Respectsˡ _≈₂_ →
+                _<ₗₑₓ_ Respectsˡ _≋_
+  ×-respectsˡ eq₁ resp₁ resp₂ x≈x' (inj₁ x₁<y₁) = inj₁ (resp₁ (proj₁ x≈x') x₁<y₁)
+  ×-respectsˡ eq₁ resp₁ resp₂ x≈x' (inj₂ x≈<y)  = inj₂ (trans (sym $ proj₁ x≈x') (proj₁ x≈<y)
+                                                     , (resp₂ (proj₂ x≈x') (proj₂ x≈<y)))
+    where open IsPartialEquivalence eq₁
+
   ×-respects₂ : IsEquivalence _≈₁_ →
                 _<₁_ Respects₂ _≈₁_ → _<₂_ Respects₂ _≈₂_ →
                 _<ₗₑₓ_ Respects₂ _≋_
-  ×-respects₂ eq₁ resp₁ resp₂ = respʳ , respˡ
-    where
-    open IsEquivalence eq₁
-
-    respʳ : _<ₗₑₓ_ Respectsʳ _≋_
-    respʳ y≈y′ (inj₁ x₁<y₁) = inj₁ (proj₁ resp₁ (proj₁ y≈y′) x₁<y₁)
-    respʳ y≈y′ (inj₂ x≈<y)  =
-      inj₂ ( trans (proj₁ x≈<y) (proj₁ y≈y′)
-           , proj₁ resp₂ (proj₂ y≈y′) (proj₂ x≈<y) )
-
-    respˡ : _<ₗₑₓ_ Respectsˡ _≋_
-    respˡ x≈x′ (inj₁ x₁<y₁) = inj₁ (proj₂ resp₁ (proj₁ x≈x′) x₁<y₁)
-    respˡ x≈x′ (inj₂ x≈<y)  =
-      inj₂ ( trans (sym $ proj₁ x≈x′) (proj₁ x≈<y)
-           , proj₂ resp₂ (proj₂ x≈x′) (proj₂ x≈<y) )
+  ×-respects₂ eq₁ resp₁ resp₂ = ×-respectsʳ trans (proj₁ resp₁) (proj₁ resp₂)
+                              , ×-respectsˡ isPartialEquivalence (proj₂ resp₁) (proj₂ resp₂)
+    where open IsEquivalence eq₁
```

Comment:
The `open` is no longer needed?

Reply:
I don't use modules that much, don't i need to open it to get access to `trans` and `sym`  ?

Reply:
Ah yes of course, sorry! Brain freeze moment.